huaweicloud/utils/utils.go

```diff
@@ -459,3 +459,13 @@ func FlattenResponse(resp *http.Response) (interface{}, error) {
 	}
 	return respBody, nil
 }
+
+// Reverse is a Function that reverses the order of the characters in the
+// given string.
+func Reverse(str string) string {
+	result := []byte(str)
+	for i := 0; i < len(str); i++ {
```

Comment:
it's better to use an effective way to reverse a string

Reply:
The function logic has been optimized.